hash-table/open-addressing.js

```diff
@@ -0,0 +1,129 @@
+const DEFAULT_CAPACITY = 8
+const UPPER_LOAD_FACTOR = 0.7
+const LOWER_LOAD_FACTOR = 0.25
+const GROW_SHRINK_FACTOR = 2
+
+class Node {
+  constructor(key, value, deleted = false) {
```

Comment:
`Node`'s constructor shouldn't allow initializing a node with any `deleted` value other than `false`. `deleted` property should only be changed by the `delete()` method.